README.md

```diff
@@ -130,13 +131,42 @@ loadConfig[MyClass](conf)
 ```
 
 
+## Integrating with other libraries
+
+### Internal Modules
+
+The core of PureConfig eschews unnessary dependencies. Separate modules exist to support types which are not part of the standard Scala and Java libraries.
+
+- [`pureconfig-enum`](modules/enum/) provides converters for enums generated by [julienrf's enum library](https://github.com/julienrf/enum).
```

Comment:
@ruippeixotog 
(Pedantic-level response)
Technically I think it'd be incorrect to use a semicolon in this case because each item of the list is a complete sentence. Although Henry David Thoreau might string together semi-colon delimited complete thoughts as a paragraph, in modern usage it'd be weird.
I believe that's in contrast to our other bulleted list in the README which is a series of predicate nominatives following a colon.
That said, if you're passionate about it, I'm happy to change it.